cnxarchive/tests/views/test_content.py

```diff
@@ -1166,6 +1244,7 @@ def test_extra_w_utf8_characters(self):
         self.assertEqual(self.request.response.content_type,
                          'application/json')
         output['canPublish'].sort()
+        output['books'].sort()
```

Comment:
This test started failing for no reason and I had to force the order on the list of books this page is in. Guessing it was always flaky, since the list of books is not ordered.